code/controllers/configuration.dm

```diff
@@ -919,6 +926,28 @@ var/list/gamemode_cache = list()
 				if ("fail2topic_enabled")
 					fail2topic_enabled = text2num(value)
 
+
+				if ("forum_api_path")
+					forum_api_path = value
+				if ("forum_api_key")
+					global.forum_api_key = value
+
+				if ("news_use_forum_api")
+					news_use_forum_api = TRUE
+
+				if ("news_forum_topics")
+					var/list/split = splittext(value, "=")
+					if (length(split) != 2)
+						error("Configuration error: couldn't split topic for news_forum_topics.")
+						break
+
+					var/list/authors = splittext(split[2], ",")
+					if (!text2num(split[1]) || !length(authors))
+						error("Configuration error: couldn't get valid topic ID or authors list for news_forum_topics.")
+						break
+
+					news_forum_topics[split[1]] = authors
```

Comment:
Imho that would be better stored in a json file, as this approach to get the forum topics and the respective author names seems rather hacky. (Possibly loaded by the news subsystem, as this data is used only there)

Reply:
HMMM. Touche, didn't really think about that.